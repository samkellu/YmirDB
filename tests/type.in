set a 123 33 22
set b a 123
type a
type b
type n
bye
