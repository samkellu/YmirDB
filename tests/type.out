> ok

> ok

> simple

> general

> no such key

> bye
